### ISSUE TITLE
Modules to load for a platform are spread out leading to potential inconsistecy

### ISSUE THREAD

**Issue by rsdunlapiv** [esmf-org/esmf-extended-tests#3]
The `module load` for a given platform are spread in multiple files:

```
(compweights) dunlap@cheyenne4:/glade/scratch/dunlap/ryan/esmf-extended-tests/ESMF_CompWeights/src> grep -l module *
buildESMF.pbs
buildTest.pbs
DiffWeights
runDiffWeights.pbs
runRWG.pbs
```

This should be consolidated into one place to specify modules for a platform.

**Comment by rokuingh** [comment]
@rsdunlapiv , yep I mentioned in the demo that a nice feature would be to put all of the module information into the config files. I'll try to do this with some spare time in the next week or so.

**Comment by rokuingh** [comment]
resolved with [c0e80b2](https://github.com/esmf-org/esmf-extended-tests/commit/c0e80b21fc325f442bb1a94b9ed9ef2abfe6cde6) and [cef1acd](https://github.com/esmf-org/esmf-extended-tests/commit/cef1acd786adc483b06ee49434654f92102e86b7)
